### PR TITLE
Topic page scatter updates

### PR DESCRIPTION
**Pull request by beets**
Config updates
- Flip ordering of scatter stat vars (y first, then x)
- Add support for contained_place_types (and added a tile-level override)

Scatter updates
- Removed extra top-margin (and it looks fine on scatter tool)
- Add support for configured stat var names

Equity updates
- Added health scatters

### CODE REVIEW

File: static/js/topic_page/block.tsx [hunk 1]
```diff
@@ -67,6 +67,10 @@ export function Block(props: BlockPropType): JSX.Element {
 function renderTiles(tiles: Tile[], props: BlockPropType): JSX.Element {
   const tilesJsx = tiles.map((tile) => {
     const id = randDomId();
+    const enclosedPlaceType =
+      tile.containedPlaceTypes && tile.containedPlaceTypes[props.place.types[0]]
```
Comment (chejennifer) [inline static/js/topic_page/block.tsx:71]: might want to check that props.place.types isn't an empty array before trying to access it. And same in the MainPane component

**Review by chejennifer — APPROVED** [review]
nice!

File: static/js/topic_page/block.tsx (hunk 1 above)
Comment (beets) [reply static/js/topic_page/block.tsx]: done - added fallback code in both places, and especially in topic_page.ts so we have a consistent fallback

**Review by beets — COMMENTED** [review]
thanks for the review!

**Pull request merged**